content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+## Introduction
+
+The {{< gloss >}}external control plane{{< /gloss >}} [deployment model](/docs/ops/deployment/deployment-models/#control-plane-models) enables mesh operators to install and manage mesh control planes on separate external clusters. This deployment model allows a clear separation between mesh operators and mesh admins. Istio mesh operators can run Istio control planes for mesh admins while mesh admins control the configuration of the control plane without worrying about installing or managing it.
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster with the Istio `default` profile installed in the `external-istiod` namespace. This istiod serves as the the external control plane for the second cluster.  An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the external istiod in the `external-istiod` namespace.
+
+The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides the configuration for the control plane (istiod) running in the external cluster.
+
```

Comment:
Please add a diagram

Reply:
Yes, will work on this in a follow up PR after able to add tests (see https://github.com/istio/istio.io/pull/8431)

Reply:
why not just include it now? It will be easier than having to go through another PR process, no?